make.ps1

```diff
@@ -18,6 +18,11 @@ function FindMSBuild
 	return $null
 }
 
+function UtilityNotFound
+{
+	echo "OpenRA.Utility.exe could not be found. Build the project first using the `"all`" command."
```

Comment:
Or it could return to the main menu where the user can immediately type "all" if he wants.